src/tasks/components/TaskRunsRow.tsx

```diff
@@ -85,6 +90,16 @@ class TaskRunsRow extends PureComponent<Props, State> {
     this.setState({isImportOverlayVisible: !this.state.isImportOverlayVisible})
   }
 
+  private handleRetry = () => {
+    const {retryTask, taskID, run} = this.props
+    try {
+      retryTask(taskID, run.id)
```

Comment:
this is an async function. If it isn't `await`ed, the same issue you and Sahas ran into with annotation errors will happen here.

you'll need to make `handleRetry` an async function